libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.ts

```diff
@@ -145,10 +173,34 @@ export class RadioGroupComponent implements AfterContentInit {
     if (value === this._value) return;
     this.setSelectedItem(value);
     this.valueChange.emit(value);
+
+    if (this.boundToForm) {
+      this._onChangeCallback(value);
+    }
   }
+
   // #endregion "protected" methods used by template
 
   // #region private methods
+
+  private trackTouchedState() {
+    // const focusChanges = this.radioButtons.toArray().map((each) =>
+    //   each.focusChange.pipe(
+    //     filter((focused) => !!focused),
+    //     take(1)
+    //   )
+    // );
+    // race(...focusChanges)
+    //   .pipe(takeUntil(race(this._newRadioButtonsSlotted, this._destroy$)))
+    //   .subscribe(() => this._onTouchedCallback());
+  }
+
+  private stopTrackingTouchedState() {
+    // this._newRadioButtonsSlotted.next();
+    // this._newRadioButtonsSlotted.complete();
+    // this._newRadioButtonsSlotted = new Subject();
+  }
+
```

Comment:
I haven't looked into this yet, so the commented out methods here serves as a reminder-to-self. I could use some feedback on how to track / set touched state.

Reply:
We got rid of this wicked code with simply a hostlistener on ionBlur() and then call onTouched.